src/test/java/de/mediathekview/mserver/crawler/srf/tasks/SrfSendungenOverviewPageTaskTest.java

```diff
@@ -42,9 +44,9 @@ public SrfSendungenOverviewPageTaskTest(String aHtmlFile, int aExpectedUrls) {
   }
     
   @Test
-  public void test() throws Exception {
+  public void test() throws IOException, Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException' which is a subclass of 'java.lang.Exception'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)